src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+
+  --header-height: 60px;
+  --pseudo-link-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+}
+
+.logo {
+  display: inline-block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  height: var(--header-height);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.nav__list {
+  text-transform: uppercase;
+  display: flex;
+  list-style: none;
+  padding: 0;
+}
+
+.nav__link {
+  display: flex;
+  text-decoration: none;
+  color: black;
+  position: relative;
+  font-size: 12px;
+  font-weight: 500;
+  margin-right: 20px;
```

Comment:
Using `margin-right` for spacing between navigation links is fine, but consider using a more flexible approach like `gap` on the parent flex container (`.nav__list`) for better control over spacing.